raiden-ts/src/channels/epics/block.ts

```diff
@@ -184,6 +186,37 @@ export function blockStaleEpic(
   );
 }
 
+/**
+ * Fetch settleTimeout from contract once
+ *
+ * @param action$ - Observable of RaidenActions
+ * @param state$ - Observable of RaidenStates
+ * @param deps - RaidenEpicDeps members
+ * @param deps.registryContract - TokenNetworkRegistry instance
+ * @param deps.config$ - Config observable
+ * @param deps.init$ - Observable of sync tasks
+ * @returns Observable of contractSettleTimeout actions
+ */
+export function contractSettleTimeoutEpic(
+  {}: Observable<RaidenAction>,
+  {}: Observable<RaidenState>,
+  { registryContract, config$, init$ }: RaidenEpicDeps,
+): Observable<contractSettleTimeout> {
+  let done$: AsyncSubject<null>;
+  return defer(async () => {
+    done$ = new AsyncSubject();
+    init$.next(done$);
+    return registryContract.callStatic.settle_timeout();
+  }).pipe(
+    retryWhile(intervalFromConfig(config$)),
+    map((settleTimeout) => contractSettleTimeout(settleTimeout.toNumber() /* seconds */)),
```

Comment:
This comment here is mostly to explain why `toNumber` isn't expected to fail here, as it can throw if the number doesn't fit in 2^53. Since this is seconds coming from the smartcontract, it won't throw, but yes, I think it's also in the action definition. Because of the io-ts codecs, I'm not sure that is automatically picked up by intellisense, but docs should be there

Reply:
Removed the comment :heavy_check_mark: